x-pack/plugins/ml/server/shared_services/providers/modules.ts

```diff
@@ -40,8 +41,14 @@ export function getModulesProvider({
       request: KibanaRequest,
       savedObjectsClient: SavedObjectsClientContract
     ) {
-      const hasMlCapabilities = getHasMlCapabilities(request);
+      let hasMlCapabilities: HasMlCapabilities;
+      if (request.params === 'DummyKibanaRequest') {
```

Comment:
I would just make `DummyKibanaRequest` a shared constant somewhere so someone doesn't accidentally break this in the future 😂

Reply:
Yeah... uptime is doing something similar as well. We want to be consistent while discouraging this temporary workaround... if it's used in a third place maybe we move it upstream into `getHasMlCapabilities`.